rclpy/src/rclpy/_rclpy_action.cpp

```diff
@@ -436,10 +438,11 @@ rclpy_action_create_client(
  * \return Capsule named 'rcl_action_server_t', or
  * \return NULL on failure.
  */
+ #include "clock.hpp"
 py::capsule
 rclpy_action_create_server(
   py::capsule pynode,
-  py::capsule pyclock,
+  py::object pyclock,
```

Comment:
It would be a little simpler to change `py::object pyclock` to `const rclpy::Clock & clock`, then instead of `auto clock = static_cast<rcl_clock_t *>(...)` the code can use the rcl pointer with `clock.rcl_ptr()`.